cmd/storage-rest-client.go

```diff
@@ -130,9 +128,6 @@ func (client *storageRESTClient) call(method string, values url.Values, body io.
 	}
 
 	err = toStorageErr(err)
-	if err == errDiskNotFound {
```

Comment:
One issue with this PR is that node that is down, comes back with disks in out of order will not be handled in this PR 

as pointed out by @krishnasrinivas shall be fixed in a subsequent PR to address this as well.